apps/lasp/src/lasp_plumtree_broadcast_distribution_backend.erl

```diff
@@ -233,25 +240,20 @@ update(Id, Operation, Actor) ->
     {ok, {Id, Type, Metadata, Value}} = gen_server:call(?MODULE,
                                                         {update, Id, Operation, Actor},
                                                         infinity),
-    BroadcastState = case Value of
-                         {delta, DeltaState, _MergedState} ->
-                             {delta, DeltaState};
-                         _ ->
-                             Value
-                     end,
     ReturnState = case Value of
                       {delta, _DeltaState, MergedState} ->
+                          %% No broadcasting for the delta.
                           MergedState;
                       _ ->
+                          case orddict:find(dynamic, Metadata) of
```

Comment:
I think these are nested wrong.  Don't you want to check if it's dynamic first?  Even if it's dynamic, you still want to avoid sending the deltas, too.